plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-terms-block/block.json

```diff
@@ -22,6 +22,10 @@
 		"text": {
 			"type": "string",
 			"required": false
+		},
+		"separatorVisible": {
```

Comment:
Nitpic: What if we either follow the pattern of the address fields and call this variable `showSeparator`? Or we create a more speaking version such as `isSeparatorVisible`?